ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/event-handlers.j2

```diff
@@ -39,57 +39,63 @@
 {%         if handler.trigger is arista.avd.defined("on-maintenance")
             and handler.trigger_on_maintenance.operation is arista.avd.defined
             and handler.trigger_on_maintenance.action is arista.avd.defined %}
-{%             set trigger_config =  "trigger " ~ handler.trigger ~ " " ~ handler.trigger_on_maintenance.operation %}
-{%             if handler.trigger_on_maintenance.bgp_peer is arista.avd.defined %}
-{%                 set trigger_config = trigger_config ~ " bgp " ~ handler.trigger_on_maintenance.bgp_peer %}
-{%                 if handler.trigger_on_maintenance.vrf is arista.avd.defined %}
-{%                     set trigger_config = trigger_config ~ " vrf " ~ handler.trigger_on_maintenance.vrf %}
+{%             set on_maintenance_cli = [] %}
```

Comment:
why we need this as list? shouldn't be this str

Reply:
it is a part of change in logic